packages/pigeon/lib/dart_generator.dart

```diff
@@ -107,8 +108,26 @@ String _makeGenericCastCall(TypeDeclaration type, String nullTag) {
       : '';
 }
 
-String _getArgumentName(NamedType field) =>
-    field.name.isEmpty ? 'arg' : field.name;
+/// Returns an argument name that can be used in a context where it is unlikely to collide.
```

Comment:
Nit: don't you mean "where it has the potential to collide"? The non-`Safe` version would be the one for contexts where you don't need to worry about collisions.

Reply:
Done.